governance/votes/Executive vote - August 18, 2023.md

```diff
@@ -0,0 +1,225 @@
+| TRUE NAME | [0x612f7924c367575a0edf21333d96b15f1b345a5d](https://etherscan.io/address/0x612f7924c367575a0edf21333d96b15f1b345a5d) | 29.76 |
+| UPMaker | [0xbb819df169670dc71a16f58f55956fe642cc6bcd](https://etherscan.io/address/0xbb819df169670dc71a16f58f55956fe642cc6bcd) | 29.76 |
+| vigilant | [0x2474937cB55500601BCCE9f4cb0A0A72Dc226F61](https://etherscan.io/address/0x2474937cB55500601BCCE9f4cb0A0A72Dc226F61) | 29.76 |
+| WBC | [0xeBcE83e491947aDB1396Ee7E55d3c81414fB0D47](https://etherscan.io/address/0xeBcE83e491947aDB1396Ee7E55d3c81414fB0D47) | 14.82 |
+|PALC| [0x78Deac4F87BD8007b9cb56B8d53889ed5374e83A](https://etherscan.io/address/0x78Deac4F87BD8007b9cb56B8d53889ed5374e83A) | 13.89 |
+|Navigator|[0x11406a9CC2e37425F15f920F494A51133ac93072](https://etherscan.io/address/0x11406a9CC2e37425F15f920F494A51133ac93072) | 11.24 |
+| PBG | [0x8D4df847dB7FfE0B46AF084fE031F7691C6478c2](https://etherscan.io/address/0x8D4df847dB7FfE0B46AF084fE031F7691C6478c2) | 9.92 |
+|VoteWizard|[0x9E72629dF4fcaA2c2F5813FbbDc55064345431b1](https://etherscan.io/address/0x9E72629dF4fcaA2c2F5813FbbDc55064345431b1) | 9.92 |
+| Libertas | [0xE1eBfFa01883EF2b4A9f59b587fFf1a5B44dbb2f](https://etherscan.io/address/0xE1eBfFa01883EF2b4A9f59b587fFf1a5B44dbb2f) | 9.92 |
+|Harmony|[0xF4704Aa4Ad22cAA2A3Dd7A7C529B4C32f7A421F2](https://etherscan.io/address/0xF4704Aa4Ad22cAA2A3Dd7A7C529B4C32f7A421F2) | 8.93|
+| JAG | [0x58d1ec57e4294e4fe650d1cb12b96ae34349556f](https://etherscan.io/address/0x58d1ec57e4294e4fe650d1cb12b96ae34349556f) | 7.61 |
+| Cloaky | [0x869b6d5d8FA7f4FFdaCA4D23FFE0735c5eD1F818](https://etherscan.io/address/0x869b6d5d8FA7f4FFdaCA4D23FFE0735c5eD1F818) | 4.30 |
+| Skynet | [0xd4d1A446cD5976a11bd32D3e815A9F85FED2F9F3](https://etherscan.io/address/0xd4d1A446cD5976a11bd32D3e815A9F85FED2F9F3) | 3.64 |
+
+### Direct Deposit Module (D3M) Parameter Housekeeping
+
+As per this [forum post](https://forum.makerdao.com/t/notice-of-executive-vote-date-change-and-housekeeping-changes/21613), the following housekeeping changes to the now defunct D3Ms will be made if this executive proposal passes.
+
+#### Aave D3M
+
+- Remove DIRECT-AAVEV2-DAI from the [DC-IAM (autoline)](https://manual.makerdao.com/module-index/module-dciam).
+- Set the DIRECT-AAVEV2-DAI [Debt Ceiling](https://manual.makerdao.com/parameter-index/vault-risk/param-debt-ceiling) to **0 DAI**.
+
+#### Compound D3M
+
+- Remove DIRECT-COMPV2-DAI from the [DC-IAM (autoline)](https://manual.makerdao.com/module-index/module-dciam).
+- Set the DIRECT-COMPV2-DAI [Debt Ceiling](https://manual.makerdao.com/parameter-index/vault-risk/param-debt-ceiling) to **0 DAI**.
+
+For completeness, the [Global Debt Ceiling (`Line`)](https://manual.makerdao.com/parameter-index/core/param-global-debt-ceiling) will be reduced by as part of this process to account for the removal of the Debt Ceiling of these collaterals.
```

Comment:
Missing amount the Global DC is reduced by

Reply:
intentional because it's being calculated as part of the spell code